packages/react-imready/src/react-imready/useImReady.tsx

```diff
@@ -1,6 +1,8 @@
-import { MutableRefObject, Ref, RefCallback, useEffect, useState } from "react";
-import ImReady from "@egjs/imready";
-import { ImReadyProps, ImReadyValue } from "./types";
+import { ImReadyHooksProps, REACTIVE_IMREADY } from "@egjs/imready";
```

Comment:
Use the name Reactive instead of Hooks.

Reply:
I used the existing type, but the name you suggested `ImReadyReactiveProps` seems more appropriate.
Would it be better to newly declare `ImReadyReactiveProps` with the same structure as the existing `ImReadyHooksProps` and deprecate the existing `ImReadyHooksProps`?